content/projects/flamelink.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Flamelink 🔥🔗
+homepage: https://flamelink.io
+twitter: FlamelinkCMS
+opensource: "No"
+typeofcms: "Realtime Headless"
+supportedgenerators:
+  - All
+description: Flamelink is an easy to use, headless CMS specifically developed to add & manage content in Firebase
+images:
+  - path: /images/flamelink-slide.png
+---
+
+## Flamelink CMS
+
+[Flamelink](https://flamelink.io) is a realtime, headless CMS for Firebase. Flamelink offers Coders and Content Managers an easy-to-use interface to add, manage and update content seamlessly integrated with their Firebase project. This lets Developers do what they do best — write white-hot code, while Content Managers can play around with content to their hearts desire.
+
+<img src="/images/flamelink-navigation.png" title="Flamelink Navigation">
+
+## Powered by Google's Firebase
+
+It's fast, secure, scalable and reliable. Flamelink leverages the power and features of Firebase to create a Headless, API Driven CMS to effortlessly manage your content. It's easy to use, so it's perfect for content managers and coders alike.
+
+## Customized Content
+
+<img src="/images/flamelink-bro.png" title="Flamelink: Do you even add and manage content, bro?">
```

Comment:
This image takes a lot of space, but distracts from the actual description of the CMS, should be removed.